### PR TITLE
replace high power cells with high power cell+

### PR DESCRIPTION
**Pull request by Evankhell561**
## What Does This PR Do
change all high power cells  with a 15,000 max charge with high power cell+ on the map

## Changelog
:cl:Evankhell
tweak: replace high power cells with high power cell+
/:cl:

### CODE REVIEW

**Comment by datlo** [comment]
Please explain what are you exactly doing and why you are doing it, please

**Comment by variableundefined** [comment]
If I have to hazard a guess, it's because high power cell is defined as a cell with 10000 max_charge, while the one with 15000 charge is high/plus.

This reduces the amount of snowflake varedit on the map.

That being said you should probably explain *why*

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -84452,13 +84443,11 @@
 /obj/item/rcd_ammo,
 /obj/item/rcd_ammo,
 /obj/item/rcd_ammo,
-/obj/item/stock_parts/cell/high{
-	charge = 100;
-	maxcharge = 15000
+/obj/item/stock_parts/cell/high/plus{
+	charge = 100
```
Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: this shouldn't exist either.

**Comment by Evankhell561** [comment]
> If I have to hazard a guess, it's because high power cell is defined as a cell with 10000 max_charge, while the one with 15000 charge is high/plus.
> This reduces the amount of snowflake varedit on the map.
Yes, it is the reason. Also, players should know what the maximum charge is only by the name of the item.

**Pull request merged**